src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,11 @@
-export const UserInfo = () => {};
+import { User } from '../../types/User';
+
+type Props = {
+  user: User | null,
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => (
+  <a className="UserInfo" href={`mailto:${user?.email}`}>
+    {user?.name}
```

Comment:
The code can potentially render null as a text inside 'a' tag if user is null. Here, it should be handled properly.